pam/pam-module-transaction-dummy.go

```diff
@@ -0,0 +1,295 @@
+package main
+
+import "C"
```

Comment:
There is no C code defined above this import, so we can remove this

Reply:
Sadly we need to have it for `getBinaryFromPointer`.